test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -318,6 +318,32 @@ Describe "TabCompletion" -Tags CI {
             $res.CompletionMatches | Should -HaveCount 1
             $res.CompletionMatches[0].CompletionText | Should -BeExactly $afterTab
         }
+
+        It "Test case insensitive <type> path" -Skip:(!$IsLinux) -TestCases @(
+            @{ type = "File"     ; beforeTab = "Get-Content f" },
+            @{ type = "Directory"; beforeTab = "cd f" }
```

Comment:
Should we have a test where we have both files and directories at the same time?

Reply:
Ok, I misunderstood your previous comment.  So you want to verify that:

```none
[file] foo
[directory] Foo
```

Hasn't impacted the current behavior?